Makefile

```diff
@@ -39,9 +39,24 @@ jsdocs:
 	#Output the final documentation link
 	echo "To read the documentation, click here : file://"${DIR}"/jsdoc/ownCloud.html"
 
+browser:
+	sed -i "s/require('fs/require('fs-web/g" owncloud/*.js
```

Comment:
Why so?
Plus, I found many other solutions for the same, but couldn't find any for webPack.